pkg/cmd/completion/completion.go

```diff
@@ -16,24 +16,24 @@ package completion
 
 import (
 	"os"
-
+	
 	"github.com/spf13/cobra"
 )
 
 const (
-	desc = `This command prints shell completion code which must be evaluated to provide
-interactive completion
+	desc = `Use to generate argument completion script 
+for one of the supported shells below. Next, invoke the script, and 
+then have interactive tab completion. 
 
 Supported Shells:
 	- bash
 	- zsh
 	- fish
 	- powershell
 `
-	eg = `To load completions:
-
+	examples = `
 Bash:
-
+# Execute the following once:
 $ source <(tkn completion bash)
 
 # To load completions for each session, execute once:
```

Comment:
nit: We probably want to mention that this needs to be executed as root to be able to write to /etc/

Reply:
@chmouel thanks for the advice for next time. As a newbie to both golang and this PR workflow. Please provide guidance for next time.